portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -23,42 +23,80 @@
 import javax.servlet.http.HttpServletResponse;
 import java.util.List; 
 import java.util.ArrayList;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
Maybe we can update the Javadoc now :)

Reply:
I added a brief description to both WelcomeServlet and CommentsServlet to explain what the doPost() and doGet() functions do